tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/connect/AMQPRedistributeClusterTest.java

```diff
@@ -42,7 +44,6 @@
 import org.apache.activemq.artemis.core.postoffice.QueueBinding;
 import org.apache.activemq.artemis.core.postoffice.impl.LocalQueueBinding;
 import org.apache.activemq.artemis.core.server.ActiveMQServer;
-import org.apache.activemq.artemis.core.server.MirrorOption;
```

Comment:
This is fine, though if you had changed it import org.apache.activemq.artemis.core.server.RoutingContext.MirrorOption instead then you wouldnt have to update all the using code to doing e.g "RoutingContext.MirrorOption.individualRoute", it would have worked as it was before.

Reply:
nice idea... done it